_cxx/envoy.mk

```diff
@@ -13,7 +13,8 @@ RSYNC_EXTRAS ?=
 
 # IF YOU MESS WITH ANY OF THESE VALUES, YOU MUST RUN `make update-base`.
```

Comment:
out of curiosity what's the difference between `make update-base` and `make generate`

Reply:
I will do a session on this, but update-base will build envoy itself using the dev envoy build container.